vhdbuilder/packer/init-variables.sh

```diff
@@ -203,7 +203,12 @@ WINDOWS_IMAGE_URL=""
 windows_servercore_image_url=""
 windows_nanoserver_image_url=""
 windows_private_packages_url=""
-windows_msi_resource_string=${WINDOWS_MSI_RESOURCE_STRING}
+
+windows_msi_resource_string=()
```

Comment:
Why an empty string is `()` instead of `""`?

Reply:
This variable is an array so the initialization is ().

Reply:
Maybe I should rename it to avoid confusion,

Reply:
Or you can construct the array in line 209 since we only use one msi for now.

Reply:
The condition could set msi as an optional choice for creating windows vm, which could help solve the issue mentioned in pr description.

Reply:
Also rename this variable to windows_msi_resource_strings to avoid confusion.